src/sentry/static/sentry/app/routes.jsx

```diff
@@ -353,6 +373,9 @@ function routes() {
         component={errorHandler(SettingsWrapper)}
       >
         <IndexRoute component={errorHandler(SettingsIndex)} />
+        <Route path="personal/" name="Personal" component={PersonalSettingsLayout}>
```

Comment:
When this rolls out to GA, are we going to switch to whatever url we're currently using (I think it's `account`?)

Reply:
This is a good question. Two answers:

1. I switched from _account_ to _personal_ because I thought it would be harder to misunderstand, but it looks like @ckj has been using _account_ and the communication is good, so I'll switch the language and naming conventions back to use _account_. 

![image](https://user-images.githubusercontent.com/435981/34058427-d7729a84-e18f-11e7-81c4-210c51bfb8e6.png)

2. That said, I think `account` should be a sub-route of `/settings` just like everything else, so maybe when everything is rtg we should just redirect `/account` urls to `/settings/account` and change the links?